static/styles/settings.css

```diff
@@ -874,15 +874,21 @@ input[type="checkbox"] {
     }
 }
 
+.edit-bot-name {
+    margin-bottom: 20px;
+}
+
+.avatar-section {
+    margin-bottom: 20px;
+}
```

Comment:
Why this change? This just adds extra space below the avatar section which isn't needed.

Reply:
it was also in manage user, so to make it consistent across all components I added that

Reply:
I think you can remove this since the extra space looks odd.